apps/shared/lib/validation/gbfs_validator.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Shared.Validation.GBFSValidator do
+  @moduledoc """
+  A module to validate GBFS feeds
+  """
+
+  defmodule Summary do
+    @moduledoc """
+    A structure holding validation results for a GBFS feed
+    """
+    defstruct has_errors: nil, errors_count: nil, version_detected: nil, version_validated: nil
```

Comment:
J'aime bien l'idée d'avoir une struct ici ; c'est un peu plus typé qu'une map, et ça documente un peu les champs.

Je serais tenté de mettre les champs en obligatoire quand c'est pertinent, avec des `@enforce_keys` (https://elixir-lang.org/getting-started/structs.html#default-values-and-required-keys).